### PR TITLE
Most recent scikit-learn results in several failed unit tests

### PR DESCRIPTION
**Pull request by it176131**
### Description

This handles the failing unit tests when `scikit-learn` is 1.3.1 or higher.

### Related issues or pull requests

#1090 
#1085 
#1087 

### Pull Request Checklist

- [x] Added a note about the modification or contribution to the `./docs/sources/CHANGELOG.md` file (if applicable)
- [x] Added appropriate unit test functions in the `./mlxtend/*/tests` directories (if applicable)
- [ ] Modify documentation in the corresponding Jupyter Notebook under `mlxtend/docs/sources/` (if applicable)
- [x] Ran `PYTHONPATH='.' pytest ./mlxtend -sv` and make sure that all unit tests pass (for small modifications, it might be sufficient to only run the specific test file, e.g., `PYTHONPATH='.' pytest ./mlxtend/classifier/tests/test_stacking_cv_classifier.py -sv`)
- [x] Checked for style issues by running `flake8 ./mlxtend`

### CODE REVIEW

**Comment by it176131** [comment]
As noted in the past few commit messages, `scikit-learn` implemented a `StackingClassifier` in 0.22 with a different API. Because of this, some of the current unit tests fail as they don't expose the necessary attributes.

@rasbt, should the `mlxtend` version of the `StackingClassifier` be replaced with the `scikit-learn` implementation? Or renamed (so to not cause confusion) and modified to pass the tests?

File: mlxtend/classifier/tests/test_stacking_classifier.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 y2 = np.c_[y, y]
 
 
+@pytest.mark.skip(reason="scikit-learn implemented a StackingClassifier in 0.22.")
```
Comment (rasbt) [inline mlxtend/classifier/tests/test_stacking_classifier.py:36]: Yes, afaik it was based on mlxtend's StackingCVClassifier. Fun fact, when I was a student, I also helped implementing the VotingClassifier in scikit-learn based on the version here in mlxtend as well as the sequential feature selection class.

**Comment by rasbt** [comment]
> @rasbt, should the mlxtend versions of the StackingClassifier and StackingRegressor be replaced with the scikit-learn implementations? Or renamed (so to not cause confusion) and modified to pass the tests?


I would prefer to keep them as the scikit-learn port is not 1:1 equivalent, and the stacking classifiers here had some advantages (and disadvantages). I can update the unit tests to add them back, no worries!

**Review by rasbt — APPROVED** [review]

**Comment by it176131** [comment]
> > @rasbt, should the mlxtend versions of the StackingClassifier and StackingRegressor be replaced with the scikit-learn implementations? Or renamed (so to not cause confusion) and modified to pass the tests?
> 
> 
> I would prefer to keep them as the scikit-learn port is not 1:1 equivalent, and the stacking classifiers here had some advantages (and disadvantages). I can update the unit tests to add them back, no worries!

Good team work! Once this is merged I'll sync my other branch and hopefully all the tests will pass 🤞

**Pull request merged**